libreddit/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_LIBREDDIT_IP
+      APP_PORT: $APP_LIBREDDIT_PORT
+
+  web:
+    image: ceramicwhite/libreddit:0.22.9@sha256:00a133da4ae8eb1b7d3ce9af0ca387e2dd929dc6b490736532b5d998a025c362
```

Comment:
@ceramicwhite Do you have the `Dockerfile`s published somewhere online?

Reply:
The images are untouched, I just pinned all the official arch images together as a list, as there are different Dockerfiles for each arch and the Images are tag separately and no official list was made.
If you inspect my list you can see the digests match the official Images on [docker hub](https://hub.docker.com/r/spikecodes/libreddit/tags)

Reply:
Ok nice 👍